modules/troubleshooting-network-observability-delay-time-records.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="nw-lokistack-rate-limits_{context}"]
+= Latency between the current time and records
+
+While collecting flows and enriching them, a latency appears between the current time and records. This applies when using Kafka on large clusters. You can configure Loki to reject old samples by editing the `lokistack-config` configmap.
+
+.Procedure
+
+. Run the following command to get the `lokistack-config` configmap in the `netobserv` namespace:
++
+[source,terminal]
+----
+$ oc get configmap lokistack-config -n netobserv -o yaml | yq '.binaryData | map_values(. | @base64d)' > binaryData.txt
+----
+
+. Edit the following parameters in the `binaryData.txt` file:
++
+[source,yaml]
+----
+spec:
+  limits_config:
+    reject_old_samples_max_age: 168h
+----
```

Comment:
We should also add a mention about `max_chunk_age`

https://github.com/netobserv/documents/blob/main/loki_config.md#delay---entry-too-far-behind-for-stream

Reply:
Addressed